src/storage/user-data.ts

```diff
@@ -59,6 +59,79 @@ function legacyPopulateSnapshotUserIds( data: UserData ): void {
 	}
 }
 
+function isVersionEqualOrGreater( version: string, compareTo: string ) {
+	const v1 = version.split( '.' ).map( Number );
+	const v2 = compareTo.split( '.' ).map( Number );
+
+	for ( let i = 0; i < Math.max( v1.length, v2.length ); i++ ) {
+		const num1 = v1[ i ] || 0;
+		const num2 = v2[ i ] || 0;
+		if ( num1 > num2 ) return true;
+		if ( num1 < num2 ) return false;
+	}
+
+	return true; // versions are equal
+}
+
+// Temporary function to include `isWpAutoUpdating` in data.sites
+// This function will be removed in a future release
+// Note: To keep it safe, we se isWpAutoUpdating to false, if something went wrong
+async function migrateWpAutoUpdating( data: UserData ) {
+	console.log('start')
+
+	let hasChanges = false;
+
+	const updatedSites = data.sites.map( ( site ) => {
+		if ( site.isWpAutoUpdating !== undefined ) {
+			return site;
+		}
+
+		hasChanges = true;
+
+		let versionFileContent = null;
+		try {
+			versionFileContent = fs.readFileSync(
+				nodePath.join( site.path, 'wp-includes', 'version.php' ),
+				'utf8'
+			);
+		} catch ( err ) {
+			return { ...site, isWpAutoUpdating: false };
+		}
+
+		const matches = versionFileContent.match( /\$wp_version\s*=\s*'([0-9a-zA-Z.-]+)'/ );
+		const version = matches?.[ 1 ];
```

Comment:
Why not calling to `getWordPressVersionFromInstallation` directly?

Reply:
Didn't see this function, thanks, I will update it